biblatex.cfg

```diff
@@ -0,0 +1,20 @@
+%Configure biblatex to fit ISO-690 UTBM's takes
+%Expect biblatex style to be authortitle as iso-authoryear force italic for title
+\DefineBibliographyStrings{french}{% Define words to prefix url and urldate
+  urlseen = {consulté le},
+  urlfrom = {disponible sur},
+}
+
+\DeclareFieldFormat{urldate}{\mkbibparens{\bibstring{urlseen}\space#1}}% Add parenthethis insted of crochet for the consult date and add urlseen as prefix
```

Comment:
```suggestion
\DeclareFieldFormat{urldate}{\mkbibparens{\bibstring{urlseen}\space#1}}% Add round brackets instead of square brackets for the consulted date and add urlseen as prefix
```